Jenkinsfile

```diff
@@ -95,18 +95,25 @@ helpers.rootLinuxNode(env, {
             }
         }
 
+        def hasGoChanges = hasChanges('go')
+        def hasJSChanges = hasChanges('shared')
```

Comment:
Just a heads up the reason why I did this checking in the shell script was so we could have finer grained control. Not just 'does this top level folder have changes'. This is fine just pointing that out